tokio/src/runtime/handle.rs

```diff
@@ -101,6 +101,32 @@ impl Handle {
         }
     }
 
+    /// Returns the flavor of the current `Runtime`.
+    ///
+    /// ```
+    /// use tokio::runtime::{Handle, Flavor};
+    ///
+    /// #[tokio::main(flavor = "current_thread")]
+    /// async fn main() {
+    ///   assert_eq!(Flavor::CurrentThread, Handle::current().get_flavor());
+    /// }
+    /// ```
+    ///
+    /// ```
+    /// use tokio::runtime::{Handle, Flavor};
+    ///
+    /// #[tokio::main(flavor = "multi_thread", worker_threads = 4)]
+    /// async fn main() {
+    ///   assert_eq!(Flavor::MultiThread, Handle::current().get_flavor());
+    /// }
+    /// ```
+    pub fn get_flavor(&self) -> Flavor {
```

Comment:
I think I prefer `RuntimeFlavor`. Also, `get_` usually isn't used in Rust.
```suggestion
    pub fn runtime_flavor(&self) -> RuntimeFlavor {
```

Reply:
Fixed.